packages/http/src/validator/__tests__/functional.spec.ts

```diff
@@ -18,6 +22,12 @@ describe('HttpValidator', () => {
       it('returns validation errors for whole request structure', async () => {
         expect(await validator.validateInput({ resource: httpOperations[2], input: BAD_INPUT })).toMatchSnapshot();
       });
+
+      describe('when all required params are provided', () => {
+        it('returns no validation errors', async () => {
+          expect(await validator.validateInput({ resource: httpOperations[0], input: GOOD_INPUT })).toMatchSnapshot();
```

Comment:
Please use `.toEqual([])` instead of `.toMatchSnapshot()` for simple expectations. E.g. this time the expectation is an empty array, so it would be way more readable when the expected result was in test code directly.